staging/src/k8s.io/client-go/tools/watch/retrywatcher.go

```diff
@@ -268,7 +271,13 @@ func (rw *RetryWatcher) receive() {
 			return
 		}
 
-		time.Sleep(retryAfter)
+		timer := rw.clk.NewTimer(retryAfter)
```

Comment:
why not `time.After`? I don't see clk used anymore (I guess it was for mocking?).

Reply:
@sttts Please see above https://github.com/kubernetes/kubernetes/pull/93709#discussion_r522432185 =)

Reply:
I don't mind removing `clk` if you think we don't need it. Just added it for "completeness" - if there will be a need to write tests for this or other things in the future. In practice it's probably not worth it trying to test this (seems like a lot of setup would be needed).

Reply:
Removed clock.